packages/core/test/acceptance/destroy_ref_spec.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {NgIf} from '@angular/common';
+import {Component, createEnvironmentInjector, DestroyRef, Directive, EnvironmentInjector, inject} from '@angular/core';
+import {TestBed} from '@angular/core/testing';
+
+describe('DestroyRef', () => {
+  describe('environnement injector', () => {
```

Comment:
```suggestion
  describe('for environment injectors', () => {
```
nit: could make this read like a sentence (and similar below)

(https://github.com/angular/angular/blob/main/docs/CODING_STANDARDS.md#testing)

Reply:
👍